src/app/introduction/activity-2/activity-2.component.html

```diff
@@ -9,5 +9,5 @@
       <p>{{language?.descriptions.bodyContent4 +' '+ email}}</p>
     </div>
   </div>
-  <button-nav [completed]="completed" nexturl="/introduction/activity-3" prevurl="/introduction/activity-1"></button-nav>
+  <button-nav [completed]="completed" nexturl="/introduction/activity-3/1" prevurl="/introduction/activity-1"></button-nav>
```

Comment:
No need to change previous url?

Reply:
@shreyans29  Not for now, because multiple levels for those activities have not been done in this PR.

Reply:
got it

Reply:
@lunayach please make note of such changes that need to be added in the next PR.
It is very easy to miss them.